pallets/runtime/tests/src/compliance_manager_test.rs

```diff
@@ -988,17 +826,16 @@ fn scope_asset_compliance() {
 }
 fn scope_asset_compliance_we() {
     // 0. Create accounts
-    let owner = AccountKeyring::Alice;
-    let owner_signed = Origin::signed(owner.public());
+    let owner = User::new(AccountKeyring::Alice);
     let cdd_signed = Origin::signed(AccountKeyring::Bob.public());
     let cdd_id = register_keyring_account(AccountKeyring::Bob).unwrap();
     let user_id = register_keyring_account(AccountKeyring::Charlie).unwrap();
     // 1. Create a token.
-    let (ticker, owner_did) = make_ticker_env(owner, vec![b'A'].into());
+    let (ticker, _token) = create_token();
```

Comment:
```suggestion
    let (ticker, _) = create_token();
```
(and in other places...)
(mostly a matter of preference to scan less code and not having to think about the bindings introduced)